packages/site/src/lib/components/shell/Header.svelte

```diff
@@ -39,11 +39,11 @@
       <i class="far fa-donate" />
       <span class="ml-1 hidden lg:inline">{$page.data.t('header.donate')}</span>
     </Button>
-    <Button href="/about" form="text" class="!hidden !lg:inline" target="_blank">
+    <Button href="/about" form="text" class="!hidden !lg:inline">
```

Comment:
@jacob-8 I need to make a comment about this: I would think that this does not need to be changed. I think you are doing this because of a misunderstanding at our last meeting. Anna explicitly was the one who requested that each of these options open in another tab, but she was complaining that on her computer her browser was opening them in another window instead. But I explained to her that she needs to change her browser settings. I think she has already solved it.

Reply:
Using target blank is good in a situation where a user is in a form and you present a link to a separate website and you want to ensure they don't lose their progress by navigating to a reference link. They should rarely be used within the same app unless you want to force that same functionality of having two things open for reference sake. For the standard use case of navigation never do this. The browser gives the user that choice if they simply middle mouse click or ctrl+click. By setting target to blank we force the user into this option and only this option. It's better to let users decide by leaving it off.